functional-samples/ai.gemini-on-device/README.md

```diff
@@ -9,12 +9,12 @@ The extension provides a chat interface using the Prompt API with Chrome's built
 ## Running this extension
 
 1. Clone this repository.
-1. Launch Chrome with the following flags:
-   `--enable-features=EnablePromptAPIForExtension --origin-trial-public-key=dRCs+TocuKkocNKa0AtZ4awrt9XKH2SQCI6o4FY6BNA=`
+1. Launch Chrome with the following flag, which makes sure the origin trial token in `manifest.json` is accepted by the browser:
```

Comment:
Should we mention here that this will go away once the OT is live?

Reply:
Good point. Done.